libs/core/src/lib/select/select.component.scss

```diff
@@ -80,6 +80,17 @@
                 }
             }
         }
+		&.fd-button--noborder {
```

Comment:
Looks good, but fed-button-noborder and splitborder are missaligned. because to indend those two
are you using tabs, but i guess we have already commented thsi before.